kiwi_lint/auto_field.py

```diff
@@ -0,0 +1,21 @@
+from pylint import interfaces, checkers
+from pylint.checkers import utils
+
+
+class AutoFieldChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'auto-field-checker'
+
+    msgs = {
+        'E5000': (
```

Comment:
Use 'R4521' as message code b/c this one is next in order.

Reply:
:+1: